integ-tests/src/main/java/com/google/testdata/TestDataHelper.java

```diff
@@ -28,12 +28,15 @@ private TestDataHelper() {}
    */
   public static void generateUsers(Connection connection, int userCount) throws SQLException {
     final String sqlDataPath = "src/main/java/com/google/sql/data/users_data.sql";
+    final String password = "test_" + nanoTime();
 
     List<String> sqlQueries = Collections.nCopies(userCount, SqlHelper.getSql(sqlDataPath));
     sqlQueries =
-        sqlQueries.stream().map(e -> e = format(e, nanoTime())).collect(Collectors.toList());
+        sqlQueries.stream().map(e -> e = format(e, nanoTime(), password)).collect(toList());
     SqlHelper.executeQueries(connection, sqlQueries);
-    logger.info(
-        format("Generated %s new Users:\n%s", sqlQueries.size(), Joiner.on("\n").join(sqlQueries)));
+
+    String message = Joiner.on("\n").join(sqlQueries);
+    message = message.replaceAll(password, "####");
```

Comment:
You could just chain this instead of reassigning `message`: `... join(...).replaceAll(...)`.

Reply:
Done.